src/components/navigation/types.ts

```diff
@@ -16,11 +16,15 @@ export interface NavigationItemProps extends Omit<BoxProps, 'css' | 'id'> {
   href?: string;
 }
 
-export interface TreeNode extends NavigationItemProps {
-  id: string;
-  title: string;
```

Comment:
these were duplicated in the extension of the interface, making it redundant and also kinda misleading